src/cittadini/00_data.tf

```
@@ -1,4 +1,10 @@
 data "azurerm_api_management" "this" {
   name                = local.apim_name
   resource_group_name = local.apim_rg
-}
\ No newline at end of file
+}
+
+data "azurerm_resource_group" "workbook_rg" {
+  name = var.monitor_resource_group_name
```

don't use a variable, it has a constant name whose environment specific portion is known
```suggestion
  name = "arc-${var.env_short}-itn-core-monitor-rg"
```

---

@umbcoppolabottazzi what do you think about this?

---

```suggestion
  name = "${local.project_core}-monitor-rg"
  
  project_core = "${var.prefix}-${var.env_short}-${var.location_short}-core"
```